secrets.tf

```diff
@@ -25,6 +25,14 @@ data "aws_secretsmanager_secret_version" "rds" {
   secret_id = "${local.config_var_prefix}rds"
 }
 
+data "aws_secretsmanager_secret_version" "admin_push_service_token" {
```

Comment:
This two secrets probably grant an update in the docs to include them (see the docs folder)

Reply:
Docs added in a661406